lib/arke/command.rb

```diff
@@ -13,9 +13,27 @@ def run!
 
     def load_configuration
       configs = YAML.load(File.open('config/variables.yaml'))
+      strategy = YAML.load_file('config/strategy.yaml')['strategy']
+
```

Comment:
I don't like this definition of drivers and stategies here.
What do you think if we will simply try smth like 
```ruby
config.strategy = "Arke::Stategies::#{strategy['type'].capitalize}".constantize
```

Reply:
If we will have convention for naming, ok

Reply:
UPD: this is ActiveSupport feature :D

Reply:
omg, really, my bad

Reply:
ok, let's keep it like this before we find something beautifull